core/src/main/java/lucee/runtime/query/caster/OtherCast.java

```diff
@@ -33,21 +35,38 @@ public OtherCast(int type) {
 
 	@Override
 	public Object toCFType(TimeZone tz, ResultSet rst, int columnIndex) throws SQLException {
-		if (type != Types.SMALLINT) return rst.getObject(columnIndex);
+		if (type != Types.SMALLINT) {
+			Object value = rst.getObject(columnIndex);
+			
+			// Drivers like Postgres like to return java.util.UUID instances instead of the string GUID 
+			if( value instanceof UUID ) {
+				return ((UUID)value).toString();
+			}
```

Comment:
The original issue indicated the problem was with UUID, but we ended up using GUID, which is also problematic. Can we do the same thing with GUID?

Reply:
@redtopia Oh yeah, I meant to ask about that.  There is no GUID type in Postres that I can see-- at least not in their unit tests that go through and test all their data types.  THere is also no such java.util.GUID class, ONLY java.util.UUID.  Can you be more specific about what exact object you're getting back in your result sets?

Reply:
@bdw429s I think you are correct actually. I wasn't sure... my team concurs.